src/app/give-help/other-form/other-form.component.scss

```diff
@@ -0,0 +1,9 @@
+.cancel-label {
```

Comment:
Może by to wrzucić do zbiorczego arkusza
<img width="607" alt="Zrzut ekranu 2022-05-23 o 11 02 41" src="https://user-images.githubusercontent.com/12514322/169784569-a22379e2-78ad-4cca-af05-31880fcf67cc.png">
<img width="607" alt="Zrzut ekranu 2022-05-23 o 11 02 41" src="https://user-images.githubusercontent.com/12514322/169784569-a22379e2-78ad-4cca-af05-31880fcf67cc.png">